config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Encass.java

```diff
@@ -198,11 +198,13 @@ public void setAnnotatedEntity(AnnotatedEntity<Encass> annotatedEntity) {
         this.annotatedEntity = annotatedEntity;
     }
 
+    @JsonIgnore
```

Comment:
Can you check adding this annotation to `AnnotableEntity` interface works? This way we won't have to add this explicitly to all the entities.

Reply:
done, it worked